parser/mpFuncCommon.cpp

```diff
@@ -856,6 +870,39 @@ MUP_NAMESPACE_START
     return new FunTimeDiff(*this);
   }
 
+  //------------------------------------------------------------------------------
+  //
+  // class FunCurrentTime
+  //
+  //------------------------------------------------------------------------------
+
+  FunCurrentTime::FunCurrentTime()
+    :ICallback(cmFUNC, _T("current_time"), -1)
+  {}
+
+  void FunCurrentTime::Eval(ptr_val_type &ret, const ptr_val_type *a_pArg, int a_iArgc)
+  {
+    if (a_iArgc != 0)
+      throw ParserError(ErrorContext(ecTOO_MANY_PARAMS, GetExprPos(), GetIdent()));
+
+    std::time_t t = std::time(0);
+    std::tm now = *std::localtime(&t);

```

Comment:
should we us gmtime here instead? i was curious, i'd guess it's better to save in the user current time instead of always utc to avoid problems

Reply:
Thr ideal would be to receive the timezone as argument. And use UTC as default. Is it possible?